dependencies.props

```diff
@@ -19,24 +19,33 @@
     <StandardCurrentRef>549ba26b2ed7c341a287c63b494e38496319802c</StandardCurrentRef>
   </PropertyGroup>
 
-  <!-- Auto-upgraded properties for each build info dependency. -->
+  <!-- Product dependency versions. -->
+  <PropertyGroup>
+    <NETStandardLibraryPackageVersion>2.0.1</NETStandardLibraryPackageVersion>
+    <NETStandardLibraryPackageId>NETStandard.Library</NETStandardLibraryPackageId>
+    <MicrosoftNETCoreRuntimeCoreCLRPackageVersion>2.0.2-servicing-25712-01</MicrosoftNETCoreRuntimeCoreCLRPackageVersion>
```

Comment:
I was under the assumption that CoreCLR was a tools and test dependency.  But I could be wrong.

Reply:
Hmm. Used in `ilasm.depproj` and `runtime.depproj`. I think you're right, changing.